conf/nutch-default.xml

```diff
@@ -548,15 +548,18 @@
 </property>
 
 <property>
-    <name>db.url.normalizers</name>
+    <name>crawldb.url.normalizers</name>
     <value>false</value>
```

Comment:
crawldb.url.normalizers is also overwritten from command-line (`updatedb ... -normalize`).